activestorage/lib/active_storage/engine.rb

```diff
@@ -30,6 +30,7 @@ class Engine < Rails::Engine # :nodoc:
     config.active_storage.analyzers = [ ActiveStorage::Analyzer::ImageAnalyzer::Vips, ActiveStorage::Analyzer::ImageAnalyzer::ImageMagick, ActiveStorage::Analyzer::VideoAnalyzer, ActiveStorage::Analyzer::AudioAnalyzer ]
     config.active_storage.paths = ActiveSupport::OrderedOptions.new
     config.active_storage.queues = ActiveSupport::InheritableOptions.new
+    config.active_storage.precompile_assets = true
```

Comment:
This config and the config added in https://github.com/rails/rails/pull/42856 should be documented. Could you add that?

It would go here https://github.com/rails/rails/blob/main/guides/source/configuring.md

Reply:
@ghiculescu I added some doc, tell me if it's OK